airflow-cluster/base-images/airflow/scripts/entrypoint.sh

```diff
@@ -20,6 +20,7 @@ TRY_LOOP="20"
 : "${POSTGRES_DATABASE:="airflow"}"
 : "${START_FILE_BEAT:="1"}"
 : "${AIRFLOW_COMMAND:=}"
+: "${AIRFLOW_HOSTS_TO_TRUST:=}"
```

Comment:
Because depending on your openshift host, you could have the problem or not. And host are different for each deployment

Reply:
Having a value should be an exception

Reply:
but having as default the openshift host it would solve the main issue by default, always, right? have you tried on the cluster the issue is not happening if that is set? if it does not affect then I would suggest it already comes so users do not need to define it for the openshift cluster host, what do you think?